test/junit/scala/tools/nsc/interpreter/ScriptedTest.scala

```diff
@@ -0,0 +1,46 @@
+package scala.tools.nsc
+package interpreter
+
+import org.junit._, Assert._, runner.RunWith, runners.JUnit4
+
+@RunWith(classOf[JUnit4])
+class ScriptedTest {
+  import javax.script._
+  import scala.tools.nsc.interpreter.Scripted
+
+  //new ScriptEngineManager().getEngineByName("scala").asInstanceOf[ScriptEngine with Compilable]
```

Comment:
what is the reason for having this commented line? is this method of getting the script engine tested somewhere else?